src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -17,25 +17,37 @@
 
     public static final String USERGUIDE_URL =
             "https://ay2122s2-cs2103-f11-2.github.io/tp/UserGuide.html";
-    public static final String HELP_MESSAGE = "Available functions on this version: \n\n"
-            + "- add n/NAME id/STUDENT_ID c/COURSE \n-delete STUDENT_ID\n"
-            + "- find k/KEYWORD [k/MORE_KEYWORDS] f/ATTRIBUTE_FIELD\n"
-            + "- delete [INDEX]\n"
-            + "- sort s/[attribute] \n"
-            + "- edit [INDEX] [PREFIX]/[NEW VALUE]\n"
-            + "- exit\n\n"
-            + "The full guide is available on our User Guide";
-
+    private static final String HELP_MESSAGE = "Available functions on this version: \n\n";
+    private static final String ADD_FIELD = "n/NAME id/STUDENT_ID c/COURSE avail/AVAILABILITY";
+    private static final String DELETE_FIELD = "INDEX";
+    private static final String EDIT_FIELD = "INDEX [PREFIX]/[NEW VALUE]";
+    private static final String FIND_FIELD = "k/KEYWORD k/[MORE_KEYWORDS] f/[ATTRIBUTE_FIELD]";
+    private static final String FOCUS_FIELD = "INDEX";
+    private static final String SORT_FIELD = "s/ATTRIBUTE";
```

Comment:
Sorry @LeeEnEn not sure if i missed it out, will we be including commands like `view` and `schedule`?

Reply:
Yeps, I can add those in. 😄

Reply:
Thank you!!!! 😄